Resources/Prototypes/_Goobstation/Balloons/balloons.yml

```
@@ -0,0 +1,376 @@
+    - trigger:
+        !type:DamageTrigger
+        damage: 1           # balloon health
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          path: /Audio/_Goobstation/BalloonEffect/balloon_pop.ogg
+          params:
+            variation: 0.125
+            volume: -4
+      - !type:SpawnEntitiesBehavior # Balloon Loot and spawn
+         spawn:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 9

(indentation)